Makefile

```
@@ -0,0 +1,17 @@
+BINPATH ?= build
+
```

Health check will require these
```suggestion
BUILD_TIME=$(shell date +%s)
GIT_COMMIT=$(shell git rev-parse HEAD)
VERSION ?= $(shell git tag --points-at HEAD | grep ^v | head -n 1)
```

---

👍